src/main/webapp/js/stats-page-loader.js

```diff
@@ -0,0 +1,27 @@
+// Fetch stats and display them in the page.
+  function fetchStats() {
+    const url = '/stats';
+    fetch(url).then((response) => {
+      return response.json();
+    }).then((stats) => {
+      const statsContainer = document.getElementById('stats-container');
```

Comment:
Regarding debugging why stats.html does not show the right value, a good debugging tool would be print statements. Adding some `console.log()` statements here would give you insight as to what is happening.

For example:
```suggestion
      console.log(stats);
      const statsContainer = document.getElementById('stats-container');
```
would help you understand what the stats JS object looks like and if you are handling if correctly. You can see these print statements in the chrome developer tools console.

Remember to remove these print statements after you are done debugging.